packages/app/src/cli/validators/extensions/ui.ts

```diff
@@ -29,12 +21,46 @@ export function validateUIExtensions(extensions: UIExtension[]) {
 
 export function validateWebPixelConfig(extension: UIExtension) {
   if (!extension.configuration.runtimeContext) {
-    throw WebPixelConfigError('runtime_context')
+    throw new error.Abort(
+      `The Web Pixel Extension configuration is missing the key "runtime_context"`,
+      `Please update your shopify.ui.extension.toml to include a valid "runtime_context"`,
+    )
   }
   if (extension.configuration.configuration) {
-    throw DeprecatedPropertyError('configuration', 'settings')
+    throw new error.Abort(
+      `The property "configuration" is deprecated and no longer supported.`,
+      `It has been replaced by "settings".`,
+    )
   }
   if (!extension.configuration.settings) {
-    throw WebPixelConfigError('settings')
+    throw new error.Abort(
+      `The Web Pixel Extension configuration is missing the key "settings"`,
+      `Please update your shopify.ui.extension.toml to include a valid "settings"`,
+    )
+  }
+}
+
+const validateAuthenticatedUrls = (extension: UIExtension) => {
+  if (
+    extension.configuration.authenticatedRedirectStartUrl !== undefined &&
+    !isValidUrl(extension.configuration.authenticatedRedirectStartUrl)
+  ) {
+    throw new error.Abort(
+      `The property "authenticated_redirect_start_url" does not contain a valid URL.`,
+      `Please update your shopify.ui.extension.toml to include a valid "authenticated_redirect_start_url"`,
+    )
+  }
+
+  if (extension.configuration.authenticatedRedirectRedirectUrls !== undefined) {
+    const invalidUrls = extension.configuration.authenticatedRedirectRedirectUrls.filter((url) => !isValidUrl(url))
```

Comment:
super nit: since we're not listing the invalid urls (maybe we should?), we'd also use `Array.some` here.

Reply:
I am listing the the invalid urls, please refer to line 59 - 61 :) 
https://github.com/Shopify/cli/blob/871c86b9e52593505adc28fd6819dabdd442072f/packages/app/src/cli/validators/extensions/ui.ts#L59-L61

Although I'm only listing them for the `authenticatedRedirectRedirectUrls` property, but I think this should be okay, since the false value is obvious in a the single string property `authenticatedRedirectStartUrl`.

Reply:
ooops. missed that. Sorry!